src/modules/mc_att_control/mc_att_control_main.cpp

```diff
@@ -788,8 +788,17 @@ MulticopterAttitudeControl::control_attitude_rates(float dt)
 
 	/* angular rates error */
 	math::Vector<3> rates_err = _rates_sp - rates;
-	_att_control = _params.rate_p.emult(rates_err * tpa) + _params.rate_d.emult(_rates_prev - rates) / dt + _rates_int +
-		       _params.rate_ff.emult(_rates_sp - _rates_sp_prev) / dt;
+
+	if (_vehicle_status.system_type == 4) {
+		/* If it is a helicopter */
+		_att_control = _params.rate_p.emult(rates_err * tpa) + _params.rate_d.emult(_rates_prev - rates) / dt + _rates_int +
```

Comment:
@bartslinger We would like to adopt the helicopter approach for all vehicles. The FF term is zero for all multicopters anyway.